### PR TITLE
updated the docs for the Cloud API

### PR DESCRIPTION
**Pull request by RyuLindow**

### CODE REVIEW

File: Umbraco-Cloud/Cloud-API/index.md [hunk 1]
```diff
@@ -46,13 +46,21 @@ To use this endpoint, make a request like this:
...
-You'll notice the "plan" parameter takes either "Single" or "Studio" as valid options. The terminology used for these plans has since changed. "Single" will create a "Starter" plan whereas "Studio" a "Professional".
+You'll notice the "plan" parameter takes either "Single", "Standard" or "Studio" as valid options. The terminology used for these plans has since changed for the "Single" and "Studio" plans. "Single" will create a "Starter" plan whereas "Studio" a "Professional" one.
+:::
+
+:::note
+Do note that the /create endpoint can only be used for creating <b>v7</b> projects.
+:::
+
+:::note
+It's currently not possible to create Heartcore or Uno projects using the Cloud API. The Cloud API has not been updated for a longer time now, however, we're looking into doing a major makeover of the Cloud API.
```
Comment (sofietoft) [inline Umbraco-Cloud/Cloud-API/index.md:63]: I think these two notes could be combined to one, and then added a bit further up in the article instead.

What do you think @RyuLindow ? 🤔

File: Umbraco-Cloud/Cloud-API/index.md [hunk 2]
```diff
@@ -46,13 +46,21 @@ To use this endpoint, make a request like this:
 
     {
         "projectName": "Name of the project", // Required
-        "plan": "Single", // Required. Options are: "Single or Studio"
+        "plan": "Single", // Required. Options are: "Single, Standard or Studio" Plan names are case sensitive!
```
Comment (sofietoft) [inline Umbraco-Cloud/Cloud-API/index.md:49]:
```suggestion
        "plan": "Single", // Required. Options are: "Single, Standard or Studio". Plan names are case sensitive.
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
Thanks for updating the article @RyuLindow !

I've added a suggestion and some comments.
Let me know what you think!

File: Umbraco-Cloud/Cloud-API/index.md (hunk 1 above)
Comment (RyuLindow) [reply Umbraco-Cloud/Cloud-API/index.md:63]: Sure, I don't really mind. A bit hard to tell how notes look like without the actual preview 😐

Comment (sofietoft) [reply Umbraco-Cloud/Cloud-API/index.md:63]: There's a good overview/preview of each of them in here: https://our.umbraco.com/documentation/Contribute/Markdown-Conventions/#adding-notes-warnings-tips

Comment (RyuLindow) [reply Umbraco-Cloud/Cloud-API/index.md:63]: Okay, when I said no actual preview, I meant that specific note. I know how they look like in general 😊

Comment (RyuLindow) [reply Umbraco-Cloud/Cloud-API/index.md:63]: Just merge when you can. I don't have anything more for that 😊

**Review by sofietoft — APPROVED** [review]
Thanks for applying some of the suggestions @RyuLindow .

I've taken the liberty to merge and move the notes to the top of the article, as I feel the information needs to be provided sooner.

**Pull request merged**